src/main/java/org/breedinginsight/brapps/importer/services/processors/GermplasmProcessor.java

```diff
@@ -236,6 +239,8 @@ public void getExistingBrapiData(List<BrAPIImport> importRows, Program program)
                             breedingMethods.put(germplasm.getBreedingMethod(), breedingMethodResults.get(0));
                             breedingMethod = breedingMethods.get(germplasm.getBreedingMethod());
                         } else {
+                            ValidationError ve = new ValidationError("Breeding Method", badBreedMethodsMsg, HttpStatus.UNPROCESSABLE_ENTITY);
+                            validationErrors.addError(i+1, ve );
```

Comment:
It might be better to use something like the trait importer uses to get the row number and make a method to get the row number so we don't have to repeat this logic in all the import processors. Here is an example from the TraitFileParser:

```
validationErrors.addError(traitValidatorError.getRowNumber(i), error);
```